dev/VSIX/Extension/Cs/Dev16/WindowsAppSDK.Cs.Extension.Dev16.csproj

```diff
@@ -163,6 +164,62 @@
       <Link>Resources\VSPackage.resx</Link>
       <Generator></Generator>
     </EmbeddedResource>
+    <EmbeddedResource Include="$(ExtensionDir)Cs\Common\cs-CZ\VSPackage.cs-CZ.resx">
```

Comment:
who decides these 14 languages? @axelandrejs

Reply:
This are the languages visual studio is localized to.